src/django_components/app_settings.py

```diff
@@ -750,9 +750,10 @@ def _prepare_extensions(self, new_settings: ComponentsSettings) -> List["Compone
         )
 
         # Prepend built-in extensions
+        from django_components.extensions.defaults import DefaultsExtension
         from django_components.extensions.view import ViewExtension
 
-        extensions = [ViewExtension] + list(extensions)
+        extensions = [DefaultsExtension, ViewExtension] + list(extensions)
```

Comment:
I've implemented the feature as an extension that modifies the given kwargs. Ngl, it's very satisfying that this extensions API can be used even for features like this :D